lib/image_dart_transformer.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:barback/barback.dart';
+import 'package:path/path.dart';
```

Comment:
nit: can you `show url`? (I find it a bit puzzling otherwise / I'm not used to it)